requirements.txt

```diff
@@ -0,0 +1,7 @@
+Django==3.0.4
+django-cors-headers==3.2.1
+gunicorn==20.0.4
+matplotlib==3.2.0
+mysqlclient==1.4.6
+numpy==1.18.1
+scipy==1.4.1
```

Comment:
Where do we use these?

Reply:
They were to be used in case we make SymPy Live more generic. I think I should remove them and address that issue separately.